core/modules/system/system.module

```diff
@@ -1847,6 +1847,7 @@ function system_library_info() {
     'website' => 'http://jqueryui.com/effect/',
     'version' => $libraries['ui']['version'],
     'js' => array(
+      'core/misc/ui/vendor/jquery-color/jquery.color.js'  => array('group' => JS_LIBRARY, 'weight' => -9),
```

Comment:
What is the reasoning for adding this under `$libraries['effects']`?

Reply:
This code used to be part of the effects, but now ship as an extra file.
Effects still depend on it, so it makes sense to keep it combined.
See also [this comment](https://github.com/backdrop/backdrop-issues/issues/5401#issuecomment-997176747)

Reply:
I think this should be moved to its own library under the name `jquery.color` and then be separately placed directly in `/misc/` like other jQuery libraries. Because jQuery Color is an entirely separate project with a completely different version number and possibly release cycle (see https://github.com/jquery/jquery-color).

Then this `effects` library should depend on the new `jquery.color` library.

Reply:
This was fixed with [`07c4693` (#3868)](https://github.com/backdrop/backdrop/pull/3868/commits/07c46932ca4b021b191ea088c841447cb7bfd4f0)

Reply:
...actually no. @indigoxela you seem to have missed this suggestion:

> ...be separately placed directly in `/misc/` like other jQuery libraries.